contracts/deploy/20_malicious_token.ts

```diff
@@ -1,14 +0,0 @@
-import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
@philippecamacho I think if we remove this file we also shouldn't commit `MaliciousToken.json` below.

Reply:
Yes the script has been removed in e63afd8ccbc4d2fc9d7e25397e410ef7350a2e35, but this was after deploying the contract.